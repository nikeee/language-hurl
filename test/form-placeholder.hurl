POST https://example.org/contact
[Form]
default: false
token: {{token}}
email: john.doe@rookie.org
number: 33611223344
