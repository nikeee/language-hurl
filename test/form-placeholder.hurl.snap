>POST https://example.org/contact
#^^^^ source.hurl keyword.control.method.hurl
#    ^ source.hurl
#     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hurl string.quoted.url.hurl string.quoted.url.hurl
>[Form]
#^^^^^^ source.hurl entity.name.section.hurl
>default: false
#^^^^^^^ source.hurl entity.name.tag.header.hurl entity.name.tag.header.hurl
#       ^ source.hurl entity.name.tag.header.hurl punctuation.separator.key-value.hurl
#        ^ source.hurl entity.name.tag.header.hurl
#         ^^^^^ source.hurl entity.name.tag.header.hurl
>token: {{token}}
#^^^^^ source.hurl entity.name.tag.header.hurl entity.name.tag.header.hurl
#     ^ source.hurl entity.name.tag.header.hurl punctuation.separator.key-value.hurl
#      ^ source.hurl entity.name.tag.header.hurl
#       ^^ source.hurl entity.name.tag.header.hurl variable.other.placeholder.hurl
#         ^^^^^ source.hurl entity.name.tag.header.hurl variable.other.placeholder.hurl
#              ^^ source.hurl entity.name.tag.header.hurl variable.other.placeholder.hurl
>email: john.doe@rookie.org
#^^^^^ source.hurl entity.name.tag.header.hurl entity.name.tag.header.hurl
#     ^ source.hurl entity.name.tag.header.hurl punctuation.separator.key-value.hurl
#      ^ source.hurl entity.name.tag.header.hurl
#       ^^^^^^^^^^^^^^^^^^^ source.hurl entity.name.tag.header.hurl
>number: 33611223344
#^^^^^^ source.hurl entity.name.tag.header.hurl entity.name.tag.header.hurl
#      ^ source.hurl entity.name.tag.header.hurl punctuation.separator.key-value.hurl
#       ^ source.hurl entity.name.tag.header.hurl
#        ^^^^^^^^^^^ source.hurl entity.name.tag.header.hurl
>